inertia/http.py

```diff
@@ -53,11 +56,19 @@ def render_ssr():
     })
 
   def page_data():
+    encrypt_history = getattr(request, INERTIA_REQUEST_ENCRYPT_HISTORY, None)
+    if encrypt_history is None:
+      encrypt_history = settings.INERTIA_ENCRYPT_HISTORY
+
+    clear_history = request.session.pop(INERTIA_SESSION_CLEAR_HISTORY, False)
+
     return {
       'component': component,
       'props': build_props(),
       'url': request.build_absolute_uri(),
       'version': settings.INERTIA_VERSION,
+      'encryptHistory': bool(encrypt_history),
```

Comment:
Should we cast this or should we raise if a we get a non-bool here?

Reply:
I don't have a very strong opinion on this one. I have added a separate commit for this to implement your suggestion.  Let me know what you think is the best approach.